stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -185,37 +184,38 @@ class FPDF:
         format: _Format | tuple[float, float] = ...,
         same: bool = ...,
         duration: int = ...,
-        transition: Any | None = ...,
+        transition: Incomplete | None = ...,
     ) -> None: ...
     def header(self) -> None: ...
     def footer(self) -> None: ...
     def page_no(self) -> int: ...
-    def set_draw_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def set_fill_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def set_text_color(self, r, g: int = ..., b: int = ...) -> None: ...
-    def get_string_width(self, s, normalized: bool = ..., markdown: bool = ...): ...
+    def set_draw_color(self, r: int, g: int = ..., b: int = ...) -> None: ...
```

Comment:
The `draw_color` attribute is currently typed as being `str`, but that doesn't look right: https://github.com/PyFPDF/fpdf2/blob/48f837963b7ae1956a0e5b23815063d23c1964f5/fpdf/fpdf.py#L923

Same with `fill_color` and `text_color`

Reply:
It's correct on 2.4.6. We can change with #8173. There are a couple other things like that